shell.nix

```diff
@@ -2,15 +2,14 @@
 
 let
   inherit (pkgs) stdenv;
-  hpkgs = pkgs.haskell.packages.ghc863;
-  ghc = hpkgs.ghcWithHoogle (hpkgs: with hpkgs; [
+  ghc = pkgs.haskellPackages.ghcWithHoogle (hpkgs: with hpkgs; [
```

Comment:
is it possible to use `(import ./default.nix).env` here instead of repeating the packages?